linera-sdk/wit/base-runtime-api.wit

```diff
@@ -64,10 +64,11 @@ interface base-runtime-api {
         height: block-height,
         index: u32,
     }
-    
+
     record bytecode-id {
         contract-blob-hash: crypto-hash,
         service-blob-hash: crypto-hash,
+        vm-runtime: vm-runtime,
     }
 
```

Comment:
@afck @jvff The coupling between the SDK and the details of runtime VMs worries me a little. Could we use abstract bytes for BytecodeId inside the SDK.